cmd/ddev/cmd/sequelpro.go

```diff
@@ -46,9 +45,7 @@ func handleSequelProCommand(appLocation string) (string, error) {
 		return "", err
 	}
 
-	nameContainer := fmt.Sprintf("%s-db", app.ContainerName())
-
-	if !dockerutil.IsRunning(nameContainer) {
+	if app.SiteStatus() != "running" {
```

Comment:
These types of assertions should probably be constants everywhere so they could be changed globally with ease. I'm not saying we need to start that here, but it's time...